README.md

```diff
@@ -133,11 +133,13 @@ egjs has the dependencies for the following libraries:
 > Except jQuery, library dependencies may vary depending on modules. For more information, see API documentation.
 
 When add dependency library have to use `yarn`. 
-> yarn add: adds a package to use in your current package.
-> yarn init: initializes the development of a package.
-> yarn install: installs all the dependencies defined in a package.json file.
-> yarn publish: publishes a package to a package manager.
-> yarn remove: removes an unused package from your current package.
+```
+yarn add: adds a package to use in your current package.
+yarn init: initializes the development of a package.
+yarn install: installs all the dependencies defined in a package.json file.
+yarn publish: publishes a package to a package manager.
+yarn remove: removes an unused package from your current package.
+```
```

Comment:
IMO, it's not necessarily force use yarn and also I'm in doubt put all yarn's commands descriptions there.
Just notify to users that we additionally support yarn also.

So, just in the process of dependency installation, adding `yarn` info will be enough.

------------

2. Clone the repository and install dependencies
```bash
...
# Install the node dependencies modules.
# using npm client
$ npm install 

# or yarn
$ yarn
```